### DISCUSSION TITLE
【一体化方案】为什么一体化方案正在逐步停止维护？

### DISCUSSION THREAD

**Discussion by cvpv** [midwayjs/midway#3753] [General / 闲聊]
文档写道：

> 一体化方案将逐步停止维护，已有项目可以继续使用，新建项目请谨慎选择。
https://www.midwayjs.org/docs/hooks/intro

为什么该方案正在逐步停止维护呢？是否有来龙去脉，我很感兴趣

**Comment by cvpv** [comment]
另外看到社区有人说：

> Midway 之前的一体化，是在服务端之上，往前探索了一阵子，我们发现 CSR/SSR 不是 Midway 擅长的领域，所以目前一体化是停滞的，后面我们所谓的一体化，更多是依赖现有的社区能力（next 等），和 Midway 的能力结合起来，会提供类似 @midwayjs/nextjs 类似的组件将 SSR 框架集成进来，不会过度封装。通过组合的形式会让前后端开发更顺畅，这样也能复用各自的能力。
https://github.com/midwayjs/midway/discussions/3641#discussioncomment-8822764

**Reply by czy88840616** [reply]
这不是社区有人说，这就是我说的

**Reply by cvpv** [reply]
请问为什么逐步停止呢，就是上面这些原因吗？还有其他原因吗

**Reply by czy88840616** [reply]
https://github.com/midwayjs/midway/discussions/3334 这里有描述过
